lib/libpcsc-cpp/src/utils.cpp

```diff
@@ -59,9 +59,9 @@ class UnexpectedResponseError : public Error
                                      const ResponseApdu& response, const char* file, const int line,
                                      const char* callerFunctionName) :
         Error("transmitApduWithExpectedResponse(): Unexpected response to command '"s
-              + bytes2hexstr(command.toBytes()) + "' - expected '"s
-              + bytes2hexstr(expectedResponseBytes) + "', got '"s + bytes2hexstr(response.toBytes())
-              + " in " + removeAbsolutePathPrefix(file) + ':' + std::to_string(line) + ':'
+              + bytes2hexstr(command) + "' - expected '"s + bytes2hexstr(expectedResponseBytes)
```

Comment:
Maybe we should add a `SecureCommandApdu` that outputs `***` when passed to `bytes2hexstr()`?

Reply:
Again out of scope this change. Only minimise copy-s of PIN memories. The issue already is in master branch